hack/quickstart/install.sh

```diff
@@ -45,4 +45,6 @@ helm install kargo \
   --set api.service.nodePort=31444 \
   --set api.adminAccount.passwordHash='$2a$10$Zrhhie4vLz5ygtVSaif6o.qN36jgs6vjtMBdM6yrU1FOeiAAMMxOm' \
   --set api.adminAccount.tokenSigningKey=iwishtowashmyirishwristwatch \
+  --set externalWebhooksServer.service.type=NodePort \
+  --set externalWebhooksServer.service.nodePort=31445 \
```

Comment:
The external webhooks server does not really play a role in the quickstart, but as long as we're not going out of our way to disable it, we may as well make it accessible.

(Personally, I find this quite convenient as I typically run through the quickstart prior to a release and then roll right into testing other features, which now includes webhooks.)